packages/dashboard/test/dashboard-keyboard.test.ts

```diff
@@ -93,7 +93,7 @@ describe('dashboard - keyboard interaction', () => {
     expect(spy.firstCall.args[0].detail).to.eql({ item: { id: 0 }, value: true });
   });
 
-  it('should not remove the widget on backspace', async () => {
+  (isChrome || isFirefox ? it : it.skip)('should not remove the widget on backspace', async () => {
```

Comment:
Is this still necessary after the selector fix?

Reply:
Yes. The test still fails Webkit tests on my Windows VM